app/liquid/liquid_renderer.rb

```diff
@@ -43,6 +44,15 @@ def images
 
   private
 
+  def set_locale
+    begin
+      if @page.language.present? && @page.language.code.present?
```

Comment:
hmm well it checks for presence, but it allows blank. I suppose I'll add an `allow_blank: false` to its validation.